modules/http/src/test/java/org/mule/module/http/functional/requester/HttpRequestConfigTestCase.java

```diff
@@ -40,7 +37,6 @@ protected String getConfigFile()
   public void verifyConfig() throws Exception
   {
     DefaultHttpRequesterConfig requestConfig = muleContext.getRegistry().lookupObject("requestConfig");
-    assertThat(requestConfig.getResponseBufferSize(), equalTo(RESPONSE_BUFFER));
```

Comment:
You are right, I'll change the test instead of removing that part. The streamResponse property tests depend on the buffer size so those are using it but only in the default way.